core/tests/participants.py

```diff
@@ -1,28 +1,9 @@
-from django.contrib.auth.models import Group, User
+import json
 from core.tests.base import BaseTestCase
-from core.models import Participant
-from core.models.insurer import Insurer
+from core.models import Participant, Race, Gender
 
 class ParticipantsTestCase(BaseTestCase):
-    def setUp(self):
```

Comment:
does this remove 'insurer'?  Cause I'm gonna be honest - it's pretty unnecessary.

Reply:
just in the test